subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -183,8 +216,19 @@ static void lp_cu_send_conn_param_req(struct ll_conn *conn, struct proc_ctx *ctx
 	if (!tx_alloc_is_available() || rr_get_collision(conn)) {
 		ctx->state = LP_CU_STATE_WAIT_TX_CONN_PARAM_REQ;
 	} else {
+		uint16_t event_counter = lp_event_counter(conn);
+
 		rr_set_incompat(conn, INCOMPAT_RESOLVABLE);
 
+		ctx->data.cu.reference_conn_event_count = event_counter;
+		ctx->data.cu.preferred_periodicity = 0U;
```

Comment:
I'd personally prefer putting this initialization in a inline function or perhaps move it to init of the procedure.